Objective-C/Internal/Replicator/CBLWebSocket.mm

```diff
@@ -472,8 +486,8 @@ - (void) _socketConnect: (NSString*)interface {
             });
         } else {
             int errNo = errno;
-            NSString* msg = interface ?
-            $sprintf(@"Failed to connect via the specified network interface %@ with errno %d", interface, errNo) :
+            NSString* msg = _networkInterface ?
+            $sprintf(@"Failed to connect via the specified network interface %@ with errno %d", _networkInterface, errNo) :
```

Comment:
There is no case that _networkInterace == nil here after all. Let log with the index instead of _networkInterace as this method is using the index.